ciprs_reader/parser/lines.py

```diff
@@ -17,7 +17,10 @@ class DefendentDOB(Parser):
 
     def clean(self, matches):
         """Parse and convert the date to ISO 8601 format"""
-        date = dt.datetime.strptime(matches["value"], "%m/%d/%Y").date()
+        try:
+            date = dt.datetime.strptime(matches["value"], "%m/%d/%Y").date()
+        except:
```

Comment:
```suggestion
        except ValueError:
```

I'd just recommend capturing the expected exception here.